tests/phpunit/CRM/Mailing/MailingSystemTest.php

```diff
@@ -170,7 +170,7 @@ public function testMailingReplyAutoRespond(): void {
 
     $replyComponent = $this->callAPISuccess('MailingComponent', 'get', ['id' => CRM_Mailing_PseudoConstant::defaultComponent('Reply', ''), 'sequential' => 1])['values'][0];
     $replyComponent['body_html'] .= ' {domain.address} ';
-    $replyComponent['body_txt'] = $replyComponent['body_txt'] ?? '' . ' {domain.address} ';
+    $replyComponent['body_txt'] ??= '' . ' {domain.address} ';
```

Comment:
While I think this is a faithful conversion, I think the original is wrong. Look at line 172, and looking at the history I think the intent is to append domain.address onto whatever is already there: https://github.com/civicrm/civicrm-core/commit/72c096abe12495677d4a89484ff1bec85b52c1fe#diff-e69658a8661c3d88a794b9346a61697d9549d5c277041c09861331715566b0d9R173. Since it looks like the test is checking that domain.address does NOT appear in the output (i.e. that the token got replaced), it always passes.

Otherwise these all look ok.

Reply:
Thanks @demeritcowboy I've removed from this PR since this is supposed to be only NFC and opened a new PR #28637 for that.